src/Containers/Blog.js

```diff
@@ -11,7 +11,7 @@ import { Card } from '../Components/Blog/Card'
 const GET_POSTS = gql`
 {
   repository(owner: "${config.githubUserName}", name: "${config.githubRepo}") {
-    issues(first: 100, states: OPEN, filterBy: { labels: "blog" }) {
+    issues(first: 100, states: OPEN, filterBy: { labels: "blog" }, orderBy: { direction: DESC, field: UPDATED_AT }) {
```

Comment:
Thanks for the PR. 
Change this to CREATED_AT because we need to show when the blog was created. 
UPADTED_AT key would change even after commenting on the blog

Reply:
Thanks for the Review.
I Changed updated at to created at :)